Source/Renderer/DrawCommand.js

```diff
@@ -15,6 +19,34 @@ define([
     function DrawCommand(options) {
         options = defaultValue(options, defaultValue.EMPTY_OBJECT);
 
+        this._boundingVolume = options.boundingVolume;
+        this._orientedBoundingBox = options.orientedBoundingBox;
+        this._cull = defaultValue(options.cull, true);
+        this._modelMatrix = options.modelMatrix;
+        this._primitiveType = defaultValue(options.primitiveType, PrimitiveType.TRIANGLES);
+        this._vertexArray = options.vertexArray;
+        this._count = options.count;
+        this._offset = defaultValue(options.offset, 0);
+        this._instanceCount = defaultValue(options.instanceCount, 0);
+        this._shaderProgram = options.shaderProgram;
+        this._uniformMap = options.uniformMap;
+        this._renderState = options.renderState;
+        this._framebuffer = options.framebuffer;
+        this._pass = options.pass;
+        this._executeInClosestFrustum = defaultValue(options.executeInClosestFrustum, false);
+        this._owner = options.owner;
+        this._debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+        this._debugOverlappingFrustums = 0;
+        this._dirty = true;
+
+// TODO: how to invalidate this?  Get/set for properties above?  Make some of them (e.g., vertex array) readonly?
```

Comment:
How are we going to release the resources for the command? Primitives release the resources for each command, but who releases the resources for derived commands?